PSKoans/Koans/Foundations/AboutHashtables.Koans.ps1

```diff
@@ -104,15 +103,15 @@ Describe 'Hashtables' {
         }
 
         It 'allows you to retrieve a list of keys or values' {
-            $Hashtable = @{One = 1; Two = 2; Three = 3; Four = 4}
+            $Hashtable = @{One = 1; Two = 2; Three = 3; Four = 4 }
 
             $Hashtable.Keys | Should -Be @('__', '__', '__', '__')
             $Hashtable.Values | Should -Be @( )
```

Comment:
The order of the keys here will still be different on every new session, so if you close your session and try to continue later, you will have to solve these again to move on.